docs/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml

```diff
@@ -1294,6 +1232,20 @@ sudo cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/environment.
                 </tbody>
               </tgroup>
             </informaltable></para>
+
+          <para><variablelist>
+              <varlistentry>
+                <term>NOTE: </term>
+
+                <listitem>
+                  <para>Some system V init system 'service hpcc-init' calls
+                  may still work and you can continue to use those. Be aware
+                  though most distributions are deprecating the service calls.
```

Comment:
However, you should be aware that most distributions are deprecating the service calls.